benchmark/find_tweet/rapidjson.h

```diff
@@ -0,0 +1,51 @@
+#pragma once
+
+#ifdef SIMDJSON_COMPETITION_RAPIDJSON
+
+#include "find_tweet.h"
+
+namespace find_tweet {
+
+using namespace rapidjson;
+
+struct rapidjson_base {
+  Document doc{};
+
+  bool run(Document &root, uint64_t find_id, std::string_view &text) {
+    if (root.HasParseError() || !root.IsObject()) { return false; }
+    auto statuses = root.FindMember("statuses");
+    if (statuses == root.MemberEnd() || !statuses->value.IsArray()) { return false; }
+    for (auto &tweet : statuses->value.GetArray()) {
+      if (!tweet.IsObject()) { return false; }
+      auto id = tweet.FindMember("id");
+      if (id == tweet.MemberEnd() || !id->value.IsUint64()) { return false; }
+      if (id->value.GetUint64() == find_id) {
+        auto _text = tweet.FindMember("text");
```

Comment:
I am fine with `_text` as a variable name but some people might object.

Reply:
Yeah ... I've been thinking about renaming all the result values "result," which would get rid of the need for things like this and make more of the code copy/pasteable. Probably do that when I implement sajson (the last one, I think).